tests/fixtures/mailbox_invalid.txt

```diff
@@ -62,7 +47,6 @@ ote"@iana.org
 .dot@iana.org
 dot.@iana.org
 two..dot@iana.org
-"Doug "Ace" L."@iana.org
```

Comment:
Valid address because we allow `Foo foo@bar.com` without `<>`. Equivalent to `"Doug "Ace " L."@iana.org` where `"Doug "Ace` is the display name.